### ISSUE TITLE
"An error occured" while updating match results

### ISSUE THREAD

**Issue by chrisalleng** [AlexRaubach/ListFortress#39]
While attempting to update the winner and score for this match:
https://listfortress.com/matches/74477/edit

to these values:
Player 1: Sascha Wagner
Player 1 points: 172
Player 2: Jeremy Batsch
Player 2 points: 51
Result: win
Winner: Sascha Wagner

An error is received:
We're sorry, but something went wrong.
If you are the application owner check the logs for more information.

**Comment by AlexRaubach** [comment]
Found the problem. I am using the Match class for both normal matches (that belong to tournaments) and Vassal League matches (which belong to a LeagueParticipant). In retrospect, maybe not the best decision. 

A while back I changed where someone updating a league match would be redirected to but that isn't a valid redirect for a tournament match, thus the 500 error. I'll add some logic to handle that case in the next day or two
